### PR TITLE
Added Initial Translation

### PR DESCRIPTION
**Pull request by jmnie**
1. Translation for home.yaml and about.yaml.
2. The direct translation was not used for some sentences for the purpose of creating a better context for Chinese readers. 
3. Straightforward translation for the slogan "I am one in a million" was used and a better translation is awaiting.
4. The two files were verified valid using [YAML Lint](http://www.yamllint.com/). 
5. Thanks, and it's my first time created the pull request! Excited.

### CODE REVIEW

File: Localization/zh-CN/home.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+url: "https://foldingathome.org/"
+language: "zh-CN"
+text: [
+# I AM One IN A MILLION bold text
+#"I AM One IN A MILLION",
+"我是百万分之一",
+
+# Associated text
+#"Regardless if you are already folding or haven't heard a word about it before, we need your help to reach our goal -- which is 1 million folders.",
+"无论你是否已经正在‘折叠’或者从未听说过这个词，我们需要你的帮助来完成愿景 -- 一百万个‘折叠者’"
```
Comment (kn1cht) [inline Localization/zh-CN/home.yaml]: need comma (`,`) at the end of this line

File: Localization/zh-CN/home.yaml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+#Associated square button
+#"START FOLDING NOW",
+"现在开始折叠",
+
+#Associated text with the aobve square button
+#"Hide this and watch the video.",
+"隐藏此处并观看视频"
```
Comment (kn1cht) [inline Localization/zh-CN/home.yaml]: need comma (`,`) at the end of this line

File: Localization/zh-CN/home.yaml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+"加入社区",
+
+# Associated text
+#"The Folding@Home community is spread over countless forums, websites and social media.
+#Here's a list of our official channels.
+#For tech support: ",
+"Folding@Home社区遍布于无数的论坛，网站和社交媒体。
+这是我们官方频道列表。
+技术支持：
+"
```
Comment (kn1cht) [inline Localization/zh-CN/home.yaml]: need comma (`,`) at the end of this line

File: Localization/zh-CN/home.yaml [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+url: "https://foldingathome.org/"
+language: "zh-CN"
```
Comment (kn1cht) [inline Localization/zh-CN/home.yaml:4]: As the latest home.yaml contains `title` field, please include title information.


https://github.com/FoldingCommunity/Translate/blob/6e122075c01321a6ae102e9559af64d55260436e/Localization/en-US/home.yaml#L1-L4

**Review by kn1cht — CHANGES_REQUESTED** [review]
Thanks @jmnie !
I'm not a zh-CN speaker, but I'll share several issues in YAML formatting that causing errors in the validation script.

**Review by jmnie — COMMENTED** [review]
1. Add minor changes to fix the comma error, hope it helps.
2. Added title field into the home.yaml

**Comment by jmnie** [comment]
> Thanks @jmnie !
> I'm not a zh-CN speaker, but I'll share several issues in YAML formatting that causing errors in the validation script.

Thank you for pointing out the error! It was my fault, I did not check the submitted home.yaml.

**Review by kn1cht — APPROVED** [review]

**Comment by fangfufu** [comment]
Thanks guys.

By the way, @kn1cht , you have write permission to this repository, so you can perform merge as well. :)

**Pull request merged**